editor/scene_tree_dock.cpp

```diff
@@ -1244,15 +1244,11 @@ void SceneTreeDock::_node_selected() {
 
 	Node *node = scene_tree->get_selected();
 
-	if (!node) {
-
-		editor->push_item(NULL);
+	if (!node)
 		return;
-	}
 
-	if (ScriptEditor::get_singleton()->is_visible_in_tree()) {
+	if (ScriptEditor::get_singleton()->is_visible_in_tree())
```

Comment:
We're actually going to *enforce* using braces even for blocks with a single statement, so these shouldn't be removed.

Reply:
I see. Updated!